colossalai/fx/profiler/meta_tensor.py

```diff
@@ -0,0 +1,46 @@
+import torch
+from torch.utils._pytree import tree_map, tree_flatten
+
+class MetaTensor(torch.Tensor):
+    """
+    A wrapping tensor that hacks `torch.autograd` without patching more `torch.ops.aten` ops.
+    """
+
+    _tensor: torch.Tensor
+ 
+    __slots__ = ['elem']
```

Comment:
what is this for?

Reply:
I mean `__slots__` here.

Reply:
Oops I forgot to update the attribute name to `._tensor`

Reply:
what is it for?

Reply:
The special attribute [__slots__](https://docs.python.org/3/reference/datamodel.html#slots) allows you to explicitly state which instance attributes you expect your object instances to have, with the expected results:

1. faster attribute access.
2. space savings in memory.